tests/blackbox/evaluation/src/scenario/enums.py

```diff
@@ -0,0 +1,42 @@
+from enum import StrEnum
+
+
+class TestStatus(StrEnum):
+    """Category represents enum for the category of an expectation."""
+
+    PASSED = "passed"
+    FAILED = "failed"
+    PENDING = "pending"
+
+
+class Complexity(StrEnum):
+    """Category represents enum for the category of an expectation."""
+
+    LOW = "low"
+    MEDIUM = "medium"
+    HIGH = "high"
+
+    def to_int(self) -> int:
+        if self == Complexity.LOW:
+            return 1
+        if self == Complexity.MEDIUM:
+            return 2
+        if self == Complexity.HIGH:
+            return 3
+        raise ValueError(f"Invalid complexity: {self}")
+
+
+class Category(StrEnum):
+    """Category represents enum for the category of an expectation."""
+
+    COMPLETENESS = "completeness"
+    PARTIAL_ANSWER = "partial_answer"
+    KYMA = "kyma"
+    KUBERNETES = "kubernetes"
+    OTHER = "other"
+    PROBLEM_FINDING = "problem_finding"
+    SOLUTION_FINDING = "solution_finding"
+    GENERAL_INQUIRY = "general_inquiry"
+    STEP_BY_STEP_GUIDANCE = "step-by-step_guidance"
+    DEFINITION_OR_EXPLANATION = "definition/explanation"
+    YAML = "yaml"
```

Comment:
Similar to the Complexity I would argue that shorter strings would make writing the test scenarios easier:
```suggestion
    COMPLETENESS = "completeness"
    PARTIAL_ANSWER = "partial"
    KYMA = "kyma"
    KUBERNETES = "k8s"
    OTHER = "other"
    PROBLEM_FINDING = "problem"
    SOLUTION_FINDING = "solution"
    GENERAL_INQUIRY = "general"
    STEP_BY_STEP_GUIDANCE = "step-by-step"
    DEFINITION_OR_EXPLANATION = "definition"
    YAML = "yaml"
```

We can still have a `to_string` or `__str__` or `pretty_str` function, that could return the full word, which would than use when we present the results.

Let me know what you think.

Reply:
I understand.